ResearchKit/Common/ORKAnswerFormat.h

```diff
@@ -1119,14 +1136,16 @@ Returns an initialized numeric answer format using the specified style, unit des
  @param minimum                 The minimum value to apply, or `nil` if none is specified.
  @param maximum                 The maximum value to apply, or `nil` if none is specified.
  @param maximumFractionDigits   The maximum fraction digits, or `nil` if no maximum is specified.
+ @param defaultNumericAnswer    The default numeric answer, or `nil` if no default is specified.
 
  @return An initialized numeric answer format.
  */
 - (instancetype)initWithStyle:(ORKNumericAnswerStyle)style
                          unit:(nullable NSString *)unit
                       minimum:(nullable NSNumber *)minimum
                       maximum:(nullable NSNumber *)maximum
-        maximumFractionDigits:(nullable NSNumber *)maximumFractionDigits NS_DESIGNATED_INITIALIZER;
+        maximumFractionDigits:(nullable NSNumber *)maximumFractionDigits
+         defaultNumericAnswer:(nullable NSNumber *)defaultNumericAnswer NS_DESIGNATED_INITIALIZER;
```

Comment:
Extended the designated initializer to include default numeric answer per unit test failure.

Reply:
Can you provide some details on what the failure was? 

I am not sure why we need to extend the default initializer to include defaultNumericAnswer, if this is indeed causing a failure, shouldn't we also do the same for defaultTextAnswer?

Reply:
defaultNumericAnswer was flagged by the tests as a property that went uninitialized by the default initializer but defaultTextAnswer was not. There's probably a better way to quiet this particular text and I missed it.